app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if(params[:uuid] == nil)
+
+      channels = Channel.all.map { |channel| channel.id }
+      data = JSON.pretty_generate(channels)
+      render(json: data)
```

Comment:
nit: parenthesis not needed `render json: data`

Reply:
Seems that we like to use the parenthesis here so I'll keep it as a Publishers convention